### DISCUSSION TITLE
Audit.WebApi Minimal Api support?

### DISCUSSION THREAD

**Discussion by santoror** [thepirat000/Audit.NET#676] [Q&A] [answered]
Hello,
pretty much the title, i didn't find any example/specific on the new minimal api.
Does Audit.WebApi support minimal api?
If not i would like to contribute adding support for it.

**Comment by thepirat000** [comment, marked as answer]
The middleware should work with minimal APIs, for example:

```c#
using Audit.WebApi;

var builder = WebApplication.CreateBuilder(args);
// ...
var app = builder.Build();

app.UseAuditMiddleware(config => config.IncludeHeaders());   // <-- THIS

app.MapGet("/weatherforecast", (HttpContext httpContext) =>
{
  // ...
});
```

**Comment by Patriciaukem** [comment]
Good morning,

Am an IT AUDITOR and I want to carry out an audit on our bank's API please I need help the questions I need to ask the auditors, the things I need to look out for, the requirements I need them to provide for me and also a checklist
